demosplan/DemosPlanCoreBundle/ResourceTypes/InvitedPublicAgencyResourceType.php

```
@@ -107,9 +134,56 @@ protected function getProperties(): ResourceConfigBuilderInterface
                 ->setFilterable();
         }
 
-        // todo transmit count of statements handed in by orga within Procedure
-        // todo add bool indicating this orga had an invitation sent via email within thin procedure phase
-
         return $configBuilder;
     }
+
+    /**
+     * Repository approach: Uses Statement.organisation (_o_id) to count statements submitted via draft-to-statement workflow.
+     * This counts statements where the organisation is set directly on the statement, representing
+     * statements submitted by the organisation through the draft-to-statement process.
+     */
+    private function getOriginalStatementsCountForOrga(string $orgaId): int
+    {
+        $procedure = $this->currentProcedureService->getProcedure();
+        if (null === $procedure) {
+            return 0;
+        }
+
+        try {
+            return $this->statementRepository->countDraftToStatementSubmissionsByOrganisation(
+                $procedure->getId(),
+                $orgaId
+            );
+        } catch (Exception) {
+            return 0;
+        }
```

I think we should avoid this kind of silencing exceptions.
Instead I suggest to remove the try catch and let the exception bubbling up. (enhancing dockblock appropriative:)

```
     * @throws NonUniqueResultException
     * @throws NoResultException
```